lib/installed-package-view.coffee

```diff
@@ -53,12 +53,16 @@ class InstalledPackageView extends View
       @div outlet: 'sections'
 
   initialize: (@pack, @packageManager) ->
+    @activate()
     @populate()
     @handleButtonEvents()
     @updateFileButtons()
     @checkForUpdate()
     @subscribeToPackageManager()
 
+  activate: ->
+    @pack.activateConfig() if not atom.packages.isPackageActive(@pack.name)
```

Comment:
This is on the installed-package view so its only loading the package you've opened not all of them.